src/encoded/types/quality_metric.py

```diff
@@ -213,6 +229,18 @@ class HistoneChipSeqQualityMetric(QualityMetric, CalculatedAssayTermID):
     item_type = 'histone_chipseq_quality_metric'
     schema = load_schema('encoded:schemas/histone_chipseq_quality_metric.json')
 
+    @calculated_property(schema={
+        "title": "Best FRiP score",
+        "type": "number",
+        "description": "Best fraction reads in peaks (FRiP) from peaks",
+        "comment": "Do not submit. The value is extracted from submitted FRiP scores.",
+        "notSubmittable": True,
+    })
+    def frip(self, request, Fp=None, Ft=None, F1=None, F2=None):
+        frips = list(filter(lambda x: x is not None, [Fp, Ft, F1, F2]))
```

Comment:
This is fine and concise though usually preferable to use list comprehension over filter/lambda.

```python
frips = [
    f
    for f in [Fp, Ft, F1, F2]
    if f is not None
]
```

https://stackoverflow.com/a/3013722

Reply:
👍 Good reference. Thank you! Fixed.